### PR TITLE
OSDOCS-5277: Document deprecated release note

### PR DESCRIPTION
**Pull request by dfitzmau**
[OSDOCS-5277](https://issues.redhat.com/browse/OSDOCS-5277)

Version(s):
4.13

Link to docs preview: [Deprecated VMware vSphere configuration parameters](https://57624--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-deprecated-features)

QE review:
- [ ] QE has approved this change.
<!--- QE approval is required to merge a PR except for changes that do not impact the meaning of the docs.

Additional resources: https://github.com/openshift/installer/blob/master/pkg/types/vsphere/platform.go#L35-L128

### CODE REVIEW

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -419,6 +419,24 @@ In the following tables, features are marked with the following statuses:
...
+* `platform.vsphere.username`
+* `platform.vsphere.password`
+* `platform.vsphere.datacenter`
+* `platform.vsphere.defaultDatastore`
+* `platform.vsphere.cluster`
+* `platform.vsphere.folder`
+* `platform.vsphere.resourcePool`
+* `platform.vsphere.apiVIP`
+* `platform.vsphere.ingressVIP`
+* `platform.vsphere.MachinePlatform`
```
Comment (jcpowermac) [inline release_notes/ocp-4-13-release-notes.adoc:437]: remove `MachinePlatform` that isn't deprecated

Comment (dfitzmau) [reply release_notes/ocp-4-13-release-notes.adoc]: Good catch. I removed this from the list.

Comment (dfitzmau) [reply release_notes/ocp-4-13-release-notes.adoc]: Hi @jcpowermac . I moved the note to a more suitable section. Please let me know what you think.

**Comment by jcpowermac** [comment]
/lgtm

cc: @rvanderp3

**Pull request closed without merging**